packages/login/src/modules/sagas.js

```diff
@@ -17,15 +17,30 @@ export const DEFAULT_TIMEOUT = 30
 export const textResourceSelector = (state, key) => state.intl.messages[key] || key
 export const loginSelector = state => state.login
 
-export function doLoginRequest(data) {
-  return fetch(`${__BACKEND_URL__}/nice2/login`, getOptions(data))
+export function doRequest(url, options) {
+  return new Promise(resolve => {
+    fetch(url, options)
+      .then(resp => {
+        resp.json().then(json => resolve(json))
+      })
+      .catch(e => {
+        if (console && console.error) {
```

Comment:
Yes. According to your link there is basic support since IE 8. And I checked it in IE 11 and Edge and it's available and seems to work.